File: Proiect/src/main/java/model/Organiser.java
package model;

import java.io.Serializable;

public class Organiser implements Identifiable<Integer>, Serializable {
    private Integer Id;
    private String name;
    private String password;

    public Organiser(Integer id, String name) {
        Id = id;
        this.name = name;
    }

    public Organiser(String name, String password) {
        this.name = name;
        this.password = password;
    }

    public Organiser() {
    }

    public Organiser(Integer id){
        this.Id = id;
    }

    public Organiser(String name) {
        this.name = name;
    }

    @Override
    public Integer getID() {
        return Id;
    }

    @Override
    public void setId(Integer id) {
        this.Id = id;
    }

    public String getName() {
        return name;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return Id + " " + name;
    }
}

File: Csharp/MPP/MPP/model/Organiser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MPP.model
{
    public class Organiser : Identifiable<int>
    {
        public Organiser()
        {
        }

        public Organiser(int id)
        {
            Id = id;
        }

        public Organiser(string name, string password) : this(name)
        {
            Password = password;
        }

        public Organiser(int id, string name, string password) : this(id, name)
        {
            Password = password;
        }

        public int Id { get; set; }

        public string Name
        {
            get;
            set;
        }

        public String Password
        {
            get; 
            set; 
        }

        public Organiser(int id, string name)
        {
            Id = id;
            Name = name;
        }

        public Organiser(string name)
        {
            Name = name;
        }

        public override string ToString()
        {
            return Id + " " + Name;
        }
    }
}
